common/data_refinery_common/models/models.py

```diff
@@ -54,6 +55,7 @@ def __str__(self):
     platform_accession_code = models.CharField(max_length=256, blank=True)
     platform_name = models.CharField(max_length=256, blank=True)
     technology = models.CharField(max_length=256, blank=True)
+    manufacturer = models.CharField(max_length=256, blank=True)
```

Comment:
It'd be great if you could update the serializers in api to add these two new fields

Reply:
Done